assets/app/views/_overview-deployment.html

```diff
@@ -42,11 +42,11 @@ <h3 class="h4">
       </h3>
     </div>
 
-    <div class="component meta-data">
+    <div class="component meta-data" ng-init="causes = (rc | deploymentCauses)">
```

Comment:
We're not reparsing it, though. It is now only parsed once in the filter. But I can change it.